custom-prom.md

```diff
@@ -45,10 +45,13 @@ To confirm this job is successfully scraped by Prometheus, you can view the Targ
 ![Prometheus Targets](/prom-targets.png)
 
 <a name="recording-rules"></a>
-### Recording Rules  
+### Recording Rules
+
+NOTE: There is no need to maintain additional recording rules starting in v1.90.0. This section will be removed soon!
```

Comment:
Should be "maintain" be "add"? That word choice sounds confusing to me if I'm setting up a new custom prom integration.

Reply:
Fixed